src/components/onboardingFormData/PersonalAndBillingDataSection.tsx

```diff
@@ -193,6 +208,32 @@ export default function PersonalAndBillingDataSection({
     };
   };
 
+  const validateRecipientCode = (recipientCodeStatus: string) => {
+    if (
+      canInvoice &&
+      formik.values.recipientCode &&
+      formik.values.recipientCode.length === 6 &&
+      recipientCodeStatus === 'DENIED_NO_ASSOCIATION'
+    ) {
+      formik.setFieldError(
+        'recipientCode',
+        t('onboardingFormData.billingDataSection.invalidRecipientCodeNoAssociation')
+      );
+    } else if (
+      canInvoice &&
+      formik.values.recipientCode &&
+      formik.values.recipientCode.length === 6 &&
+      recipientCodeStatus === 'DENIED_NO_BILLING'
+    ) {
+      formik.setFieldError(
+        'recipientCode',
+        t('onboardingFormData.billingDataSection.invalidRecipientCodeNoBilling')
+      );
+    } else {
+      formik.setFieldError('recipientCode', undefined);
```

Comment:
The second bullet point is not respected
https://pagopa.atlassian.net/jira/software/c/projects/SELC/boards/116?assignee=712020%3A02464b43-c51f-49ee-82bc-16dd1bd5dac5&selectedIssue=SELC-5253

Reply:
Closed change request, this is managed with other ticket: https://pagopa.atlassian.net/browse/SELC-5314